web-registry/src/styles/table.ts

```diff
@@ -0,0 +1,22 @@
+import { SxProps } from '@mui/material';
+
+export const ELLIPSIS_COLUMN_WIDTH = '125px';
+
+export const tableStyles = {
```

Comment:
Created `src/styles` so we can start to mutualize some common CSS written as `sx` prop.